app/src/main/java/com/example/dogbook/MainActivity/Models/User.java

```diff
@@ -0,0 +1,21 @@
+package com.example.dogbook.MainActivity.Models;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+@ParseClassName("User")
+public class User extends ParseUser {
+
+    public String getName() {
+        return getUsername();
+    }
+
+    public void setName() {
+        setUsername("");
+    }
```

Comment:
Do you need this setter?  It's just setting to empty and nothing passed in.  I would remove this and add it later when there's functionality or clearly denote that you're going to come back and fill this out with a TODO comment + description (and also nice if you link it to an issue)